.github/workflows/ci.yaml

```diff
@@ -35,13 +36,29 @@ jobs:
       - name: Build python package
         run: python -m build
 
-      - name: cx_Freeze
-        run: |
-          python setup_cxfreeze.py build
-          python setup_cxfreeze.py finalize
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dist-${{ matrix.os }}
+          path: dist
+
+  publish:
+    name: Publish to PyPI
+    needs: build
+    runs-on: [ubuntu-latest]
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: dist-ubuntu-latest
+          path: dist
+
+      - name: Publish package (test.pypi.org)
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
looks like this didn't work.  this check failed to run on this PR with the message: 

> pypa/gh-action-pypi-publish@release/v1 is not allowed to be used in secondlife/autobuild.